components/devtools/actors/inspector.rs

```diff
@@ -86,14 +86,26 @@ impl Actor for InspectorActor {
                 self.script_chan.send(GetRootNode(pipeline, tx)).unwrap();
                 let root_info = rx.recv().unwrap().ok_or(())?;
 
-                let root = root_info.encode(registry, false, self.script_chan.clone(), pipeline);
+                let name = match self.walker.borrow().as_ref() {
+                    Some(walker) => walker.clone(),
+                    None => registry.new_name("walker"),
+                };
```

Comment:
Can this be something like:

```suggestion
                let name = self.walker.borrow().cloned().unwrap_or_else(||  registry.new_name("walker"));
```

Reply:
Yes, that looks very nice!